e2e/atlas/helper_test.go

```diff
@@ -81,6 +84,10 @@ func getHostnameAndPort() (string, error) {
 }
 
 func deployCluster() (string, error) {
+	return deployClusterFromProject("")
+}
+
+func deployClusterFromProject(projectID string) (string, error) {
```

Comment:
[nit] couple of things here, first `deployClusterFromProject` seems unused, as in you only use it here, so it seems as an unnecessary change. I guess your goal was to create a new project for ldap test and then deploy for the new project, if you still plan to do that then this method should be `deployClusterForProject` or `deployClusterWithProjectID` and be used, I'd also change `deployCluster` to `deployClusterWithDefaultProject` to avoid confusion later on
Also non blocking this usage where and empty string defaults to the default project (`deployClusterFromProject("")`) should at least be documented, the name of the method and this assumption could be confusing later on so I'd rather have a nutral method shared by both `WithDefaultProject` and `WithProjectID` and let `WithProjectID` fail if no id is given